index.src.html

```diff
@@ -3317,27 +3317,234 @@
 Pixel Format{#pixel-format}
 ---------------------------
 Pixel formats describe the arrangement of bytes in each plane as well as the
-number and order of the planes.
-
-NOTE: This section needs work. We expect to add more pixel formats and offer
-    much more verbose definitions. For now, please see
-    <a href="http://www.fourcc.org/pixel-format/yuv-i420/">
-    http://www.fourcc.org/pixel-format/yuv-i420/</a> for a more complete
-    description.
+number and order of the planes. Each format is described in its own sub-section.
 
 <xmp class='idl'>
 enum PixelFormat {
-  "I420"
+  // 4:2:0 Y, U, V
+  "I420",
+  // 4:2:0 Y, U, V, A
+  "I420A",
+  // 4:2:2 Y, U, V
+  "I422",
+  // 4:4:4 Y, U, V
+  "I444",
+  // 4:2:0 Y, UV
+  "NV12",
+  // 32bpp RGBA
+  "RGBA",
+  // 32bpp RGBX (opaque)
+  "RGBX",
+  // 32bpp BGRA
+  "BGRA",
+  // 32bpp BGRX (opaque)
+  "BGRX",
 };
 </xmp>
 
 <dl>
   <dt><dfn enum-value for=PixelFormat>I420</dfn></dt>
   <dd>
-    Planar 4:2:0 YUV.
+    This format is composed of three distinct planes, one plane of Luma and two
+    planes of Chroma, denoted Y, U and V, and present in this order. It is also
+    often refered to as Planar YUV 4:2:0.
+
+    The U an V planes are sub-sampled horizontaly and vertically by a factor of
+    2 compared to the Y planes.
+
+    Each sample in this format is 8 bits.
+
+    There is {{VideoFrame/codedWidth}} * {{VideoFrame/codedHeight}} samples (and
+    therefore bytes) in the Y plane, arranged starting at the top left in the
+    image, in {{VideoFrame/codedHeight}} lines of {{VideoFrame/codedWidth}}
+    samples.
+
+    There is {{VideoFrame/codedWidth}} * {{VideoFrame/codedHeight}} / 4 samples
+    (and therefore bytes) in the two U and V planes, arranged starting at the
+    top left in the image, in {{VideoFrame/codedHeight}} / 2 lines of
+    {{VideoFrame/codedWidth}} / 2 samples.
+
+    The {{VideoFrame/codedWidth}} and {{VideoFrame/codedHeight}} MUST be even.
+    Similarly, the crop coordinates ({{VideoFrame/displayWidth}} and
+    {{VideoFrame/displayHeight}}) MUST be even.
   </dd>
-</dl>
+  <dt><dfn enum-value for=PixelFormat>I420A</dfn></dt>
+  <dd>
+
+    This format is composed of four distinct planes, one plane of Luma, two
+    planes of Chroma, denoted Y, U and V, and one place of alpha values, all
+    present in this order. It is also often refered to as Planar YUV 4:2:0.
```

Comment:
"also often referred to" matches what was written for I420. Should it mention alpha somewhere?